README.md

```diff
@@ -1,139 +1,152 @@
-```
-
-Now you can hit a Visual Studio Code breakpoint in a test. Try this:
-
-1. Add a breakpoint to `testGetCalendarView` in *.\tests\Functional\EventTest.php*.
-2. Run the **Listen for XDebug** configuration in VS Code.
-3. Run `.\vendor\bin\phpunit --filter testGetCalendarView` from the PowerShell terminal to run the test and hit the breakpoint.
-
-## Documentation and resources
-
-* [Documentation](https://github.com/microsoftgraph/msgraph-beta-sdk-php/blob/master/docs/index.html)
-
-* [Wiki](https://github.com/microsoftgraph/msgraph-beta-sdk-php/wiki)
-
-* [Examples](https://github.com/microsoftgraph/msgraph-beta-sdk-php/wiki/Example-calls)
-
-* [Microsoft Graph website](https://developer.microsoft.com/en-us/graph/)
-
-## Issues
-
-View or log issues on the [Issues](https://github.com/microsoftgraph/msgraph-beta-sdk-php/issues) tab in the repo.
-
-## Contribute
-
-Please read our [Contributing](https://github.com/microsoftgraph/msgraph-beta-sdk-php/blob/master/CONTRIBUTING.md) guidelines carefully for advice on how to contribute to this repo.
-
-## Copyright and license
-
-Copyright (c) Microsoft Corporation. All Rights Reserved. Licensed under the MIT [license](LICENSE).
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# Get started with the Microsoft Graph Beta SDK for PHP
+
+[![Latest Stable Version](https://poser.pugx.org/microsoft/microsoft-graph-beta/version)](https://packagist.org/packages/microsoft/microsoft-graph)
+
+## Install the SDK
+You can install the Beta PHP SDK with Composer, either run `composer require microsoft/microsoft-graph-beta`, or edit your `composer.json` file:
+```
+{
+    "require": {
+        "microsoft/microsoft-graph-beta": "^2.0.0-preview"
+    }
+}
+```
+## Get started with Microsoft Graph
+
+### 1. Register your application
+
+Register your application to use the Microsoft Graph API by following the steps at [Register an an application with the Microsoft Identity platform](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
+
+### 2. Authenticate with the Microsoft Graph service
+
+The Microsoft Graph Beta SDK for PHP does not include any default authentication implementations. The [`thephpleague/oauth2-client`](https://github.com/thephpleague/oauth2-client) library will handle the OAuth2 flow for you and provide a usable token for querying the Graph.
+
+For an integrated example on how to use Oauth2 in a Laravel application and use the Graph, see the [PHP Tutorial](https://docs.microsoft.com/en-us/graph/tutorials/php).
+
+To authenticate as an application you can use the [Guzzle HTTP client](http://docs.guzzlephp.org/en/stable/), which comes preinstalled with this library, for example like this:
```

Comment:
It would be helpful to link to this topic as the code example below assumes that the reader understands the parameters.
https://docs.microsoft.com/en-us/graph/auth-v2-service?context=graph%2Fapi%2F1.0&view=graph-rest-1.0